### PR TITLE
[LYS] Reapply styles from 46619

### PR DESCRIPTION
**Pull request by psealock**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

https://github.com/woocommerce/woocommerce/pull/46751 required a challenging conflict resolution that was best handled by reapplying the styles created in https://github.com/woocommerce/woocommerce/pull/46619 which added styles in case the theme wasn't FSE.

This PR applies those same styles and makes sure they are in the editor, template, and save function of the Coming soon block.

Screenshot 2024-04-16 at 15 06 42

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Visit the template in the site editor, confirm it looks as it should.  Appearance > Editor > Templates > Page: Coming soon
2. Visit the site in incognito mode, confirm the coming soon page is displayed properly.
5. Change the theme to a non FSE theme and confirm the coming soon page is displayed properly.
6. Repeat the above steps with different themes

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [x] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

Fixes unreleased feature

</details>

### CODE REVIEW

File: plugins/woocommerce-blocks/assets/js/blocks/coming-soon/edit.tsx [hunk 1]
```diff
@@ -21,21 +21,91 @@ export default function Edit( { attributes, setAttributes } ) {
 							setAttributes( { color: newColor } )
 						}
 						enableAlpha
-						defaultValue="#bea0f2"
+						defaultValue="${"
```
Comment (rjchow) [inline plugins/woocommerce-blocks/assets/js/blocks/coming-soon/edit.tsx]: I think there's a typo here 😅

Comment (psealock) [reply plugins/woocommerce-blocks/assets/js/blocks/coming-soon/edit.tsx]: Fixed in https://github.com/woocommerce/woocommerce/pull/46813/commits/e11bf0a51ae95b30ac0505394481ca513e930f97

**Review by rjchow — APPROVED** [review]
LGTM! Thanks for resolving this quickly

**Review by adrianduffell — APPROVED** [review]
LGTM thanks for your work on this @psealock 🚀

**Pull request merged**